### PR TITLE
[POC] visual hint when picking protected resources

### PR DESCRIPTION
**Pull request by m-vo**
see https://github.com/contao/core-bundle/pull/1594 
(now using the `FileTree` widget instead)

screenshot_20181104_095737

### CODE REVIEW

File: core-bundle/src/Resources/contao/widgets/FileTree.php [hunk 1]
```diff
@@ -233,6 +234,8 @@ public function generate()
 						continue;
 					}
 
+                    $arrClass[$objFiles->uuid] = 'type_' . $objFiles->type . (!$this->isUnprotected($objFiles->path) ? ' protected' : '');
```
Comment (richardhj) [inline core-bundle/src/Resources/contao/widgets/FileTree.php:237]: Wrong indentation here :-)

**Comment by leofeyer** [comment]
@contao/developers @frontendschlampe What do you think of the overlay? Does it make it clear for the user?

**Comment by Toflar** [comment]
I don't think there's an icon people will understand. I would expect some attention sign and when I hover it tells me that this file is not publicly available.
Something like this maybe: https://t4.ftcdn.net/jpg/01/17/85/53/240_F_117855339_u7lvoNIgnRaWDZEGX1ZlF1J7OTegHFu1.jpg

**Comment by m-vo** [comment]
I'd initially used the existing lock icon which in hindsight I liked a bit more because the root of the issue is protection (or ununprotection 😄). But a textual info should probably be in there as well.

**Comment by leofeyer** [comment]
We already have an icon that is similar to the one @Toflar suggested:

https://github.com/contao/contao/blob/master/core-bundle/src/Resources/contao/themes/flexible/icons/error_401.svg

**Comment by frontendschlampe** [comment]
I think the visible icon suggested by @m-vo is not good, because the file is not invisible, it's protected. Also the attention icon suggested by @Toflar is not the best one, because we use the exclamation mark for errors or notes.

I think we should use a lock icon - everywhere we use a lock for protected elements.

**Pull request closed without merging**